### ISSUE TITLE
Porting legacy tests from ".Flow/flow" — test case "should not be able to access values set in a flow across different flows"

### ISSUE THREAD

**Issue by Erikd-dev** [rainlanguage/rain.flow#41]
link [should not be able to access values set in a flow across different flows](https://github.com/rainlanguage/rain-protocol/blob/7f0b708db67001e7dc8122de8a901398ea92a2f3/test/Flow/Flow/flow.ts#L1604)

**Comment by Erikd-dev** [comment]
This test needs to be checked for relevance.

**Comment by thedavidmeister** [comment]
@Erikd-dev yes its relevant, need to ensure that state is namespaced correctly
